config.go

```diff
@@ -105,6 +106,13 @@ func (c *Config) Discover() error {
 		}
 	}
 
+	if len(c.Providers) == 0 {
+		return errors.New("no providers found")
```

Comment:
Yeah, your right about that... I also see some `errors.New` in the code (now that I'm looking for it :wink:), so apparently it doesn't matter that much and isn't used consistently...

Thanks!